frameworks/Scala/finagle/src/main/scala/Main.scala

```diff
@@ -13,14 +13,17 @@ import com.twitter.io.Buf
 object Main extends App {
 
   val mapper: ObjectMapper = new ObjectMapper().registerModule(DefaultScalaModule)
-  val dateFormat: DateFormat = new SimpleDateFormat("E, dd MMM yyyy HH:mm:ss z")
+
+  val dateFormat: ThreadLocal[DateFormat] = new ThreadLocal[DateFormat] {
+    override def initialValue: DateFormat = new SimpleDateFormat("E, dd MMM yyyy HH:mm:ss z")
```

Comment:
The format here should be "E, dd MMM yyyy HH:mm:ss 'GMT'". See the issue we hit and the pull-request here: https://github.com/twitter/finatra/pull/312